pkg/cmd/builder.go

```diff
@@ -116,7 +116,7 @@ func (option NamespaceFlagDisableEnvOption) ApplyToNamespaceFlagOptions(cfg *Nam
 // WithNamespaceFlag add namespace flag to the command, by default, it will also add env flag to the command
 func (builder *Builder) WithNamespaceFlag(options ...NamespaceFlagOption) *Builder {
 	cfg := newNamespaceFlagOptions(options...)
-	builder.cmd.Flags().StringP(flagNamespace, "n", "", cfg.usage)
+	builder.cmd.Flags().StringP(flagNamespace, "n", "default", cfg.usage)
```

Comment:
Why set `default` here?

Reply:
align with behavior of kubectl and other tools. No namespace means default.

Reply:
I think no explicit namespace flag will first resolve the namespace from the local config? Like you can set namespace value in `.kube/config` and kubectl will automatically switch to that.

Reply:
Oh I didn't considered that. Thanks.